docs/system/container-metrics.md

```diff
@@ -0,0 +1,99 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Container
+--->
+
+# Semantic Conventions for Container Metrics
+
+**Status**: [Experimental][DocumentStatus]
+
+## Container Metrics
+
+### Metric: `container.cpu.utilization`
+
+This metric is optional.
+
+<!-- semconv metric.container.cpu.utilization(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.cpu.utilization` | Gauge | `1` | Recent CPU utilization for the container. [1] |
+
+**[1]:** CPU usage percentage normalized by the number of CPU cores. The value range is [0.0,1.0].
+<!-- endsemconv -->
+
+<!-- semconv metric.container.cpu.utilization(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| [`container.id`](../attributes-registry/container.md) | string | Container ID. Usually a UUID, as for example used to [identify Docker containers](https://docs.docker.com/engine/reference/run/#container-identification). The UUID might be abbreviated. | `a3bf90e006b2` | Recommended |
+<!-- endsemconv -->
+
+### Metric: `container.memory.utilization`
+
+This metric is optional.
+
+<!-- semconv metric.container.memory.utilization(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.memory.utilization` | Gauge | `1` | Recent memory utilization for the container. [1] |
+
+**[1]:** Memory usage percentage. The value range is [0.0,1.0].
+<!-- endsemconv -->
+
+<!-- semconv metric.container.memory.utilization(full) -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| [`container.id`](../attributes-registry/container.md) | string | Container ID. Usually a UUID, as for example used to [identify Docker containers](https://docs.docker.com/engine/reference/run/#container-identification). The UUID might be abbreviated. | `a3bf90e006b2` | Recommended |
+<!-- endsemconv -->
+
+### Metric: `container.disk.io`
+
+This metric is optional.
+
+<!-- semconv metric.container.disk.io(metric_table) -->
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `container.disk.io` | Counter | `By` | Disk bytes for the container. [1] |
+
+**[1]:** The total number of bytes read/written successfully (aggregated from all disks).
```

Comment:
What about an IOPS metric?

Reply:
I guess "operations per second" should be another different metric if we need it?

This one provides the basic stats like disk's io bytes like what [`docker stats` provide for example](https://docs.docker.com/engine/reference/commandline/stats/#examples).

Reply:
I was referring to [CPU resource units in Kubernetes](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu). Here I can define, how much "millicpus" a container can use. If I define 500m or 0.5 and the container is using 250m, will this be reported as 50% of the 500m or 250m of a full cpu (ignoring the limit). If the latter is the case, how can I transport the limit in that case, to investigate whether the container is saturating its assigned cpu?

Reply:
I am running OpenSearch pods, which can exceed IOPS of the mounted volumes on high load. So this would be an interesting metric. But I am unsure of tooling, that could provide the information on the level, this convention is targeting.

Reply:
I see! I suggest we discuss this separately as a different metric since it's not a blocker for this PR anyway.

Reply:
Isn't IOPS just the rate per second over the `container.disk.io`? In that case, not sure why it needs to be another metric

Reply:
I would prefer `container.disk.io` in order to be consistent with https://github.com/open-telemetry/semantic-conventions/blob/705155140048f518406958bbb9b7283be4c95f26/model/metrics/system-metrics.yaml#L216-L224